Classes/Domain/Model/Book.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace TTN\Books\Domain\Model;
+
+final class Book
```

Comment:
Why the final?

Reply:
Is there any possible use case (at the moment) where this class would need to be extended? If not, I think we should declare all classes as final; if the need arises, we can always remove that keyword again.

Reply:
Yes, for XClasses.

Reply:
I don't see why a model should be XClassed. But that's probably something we need to discuss more in detail.